docs/automated-migration.md

```diff
@@ -10,50 +10,72 @@ Robolectric provides an automated migration tool to help keep your test suite up
 
 The migration tool will make changes directly to source files in your codebase, which you can review and commit to your source control system.
 
-***Before*** updating your dependencies to the new version of Robolectric:
+**Before** updating your dependencies to the new version of Robolectric:
 
 1. Make sure you're using a recent version of Gradle (4.10 or newer).
 
-2. [Configure your project](https://errorprone.info/docs/installation) to compile using Error Prone. Quick config for Gradle (usually in `app/build.gradle`):
+2. [Configure your project](https://errorprone.info/docs/installation) to integrate Error Prone. Quick config for Gradle (usually in your module's `build.gradle`/`build.gradle.kts` file):
+
+=== "Groovy"
 
     ```groovy
     plugins {
-        id "net.ltgt.errorprone" version "0.6" apply false
+        id "net.ltgt.errorprone" version "<error_prone_plugin_version>" apply false
     }
 
-    String roboMigration = System.getenv("ROBOLECTRIC_MIGRATION")
-    if (roboMigration) {
+    String robolectricMigrations = System.getenv("ROBOLECTRIC_MIGRATIONS")
+    if (robolectricMigrations) {
         apply plugin: "net.ltgt.errorprone"
 
         dependencies {
-            errorprone "com.google.errorprone:error_prone_core:2.3.2"
-            errorproneJavac "com.google.errorprone:javac:9+181-r4173-1"
```

Comment:
From what I could see, this dependency is added automatically by the Gradle plugin, in specific Java version:
https://github.com/tbroyer/gradle-errorprone-plugin/blob/e22bbca556aca14bc10e34324675877c67ad5f0e/src/main/kotlin/net/ltgt/gradle/errorprone/ErrorPronePlugin.kt#L103-L111